### ISSUE TITLE
Does renew support nginx mode ?

### ISSUE THREAD

**Issue by rahul-mnz** [acmesh-official/acme.sh#4520]
I created the cert using nginx mode which works fine but during renew this goes into standalone mode and fails to renew because of 80 port in use by nginx. Is there any workaround for this ?

**Comment by Neilpang** [comment]
show me the info 

```
acme.sh  --info  -d  youdomain.com
```

**Comment by rahul-mnz** [comment]
DOMAIN_CONF=/root/.acme.sh/<dns>/<dns>.conf
Le_Domain=<dns>
Le_Webroot=no
Le_API=https://acme.zerossl.com/v2/DV90
Le_RealCertPath=
Le_RealCACertPath=
Le_RealKeyPath=/etc/nginx/certificates/<dns>/key.pem
Le_ReloadCmd=systemctl reload nginx
Le_RealFullChainPath=/etc/nginx/certificates/<dns>/cert.pem
Le_Alt=
Le_PreHook=
Le_PostHook=
Le_RenewHook=

**Comment by neilpang** [comment]
Yes, renew supports nginx mode: `--renew` simply replays the validation mode recorded at issue time. Your conf shows `Le_Webroot=no`, which is the standalone marker -- this cert was actually issued in standalone mode (an `--issue --nginx` run records `Le_Webroot=nginx:`). Re-issue once with `acme.sh --issue --nginx -d yourdomain.com --force`, confirm the conf then shows `Le_Webroot=nginx:`, and all later renewals will use nginx mode. Closing as a configuration issue.
